docs/source/datasets/index.rst

```diff
@@ -461,4 +455,148 @@ The following columns are included in these datasets:
 Tax form: 1040
 --------------
 As with data collected from W-2 and 1099 forms, Pseudopeople will also enable the simulation of administrative records from 1040 forms, which are
-also reported to the IRS on an annual basis. This feature has not yet been implemented, so please stay tuned for more information! 
+also reported to the IRS on an annual basis. To find out more about the 1040 tax form, visit the `IRS information page <https://www.irs.gov/instructions/i1040gi>`_.
+
+Generate 1040 data with pseudopeople.generate_taxes_1040.
+
+.. todo::
+  Add link to docstring once ready.
+
+The following columns are included in these datasets:
+
+.. list-table:: **Dataset columns**
+   :header-rows: 1
+
+   * - Attribute Name
+     - Column Name
+     - Notes
+   * - Unique simulant ID
+     - :code:`simulant_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation.
+   * - Unique household ID
+     - :code:`household_id`
+     - Not affected by noise functions; intended use is "ground truth" for testing and validation; consistent across all
+       datasets. 
+   * - First name
+     - :code:`first_name`
+     - 
+   * - Middle initial
+     - :code:`middle_initial`
+     - 
+   * - Last name
+     - :code:`last_name`
+     - 
+   * - Mailing address street number
+     - :code:`mailing_address_street_number`
+     - 
+   * - Mailing address street name
+     - :code:`mailing_address_street_name`
+     - 
+   * - Mailing address unit number
+     - :code:`mailing_address_unit_number`
+     - 
+   * - Mailing address PO box
+     - :code:`mailing_address_po_box`
+     - 
+   * - Mailing address city
+     - :code:`mailing_address_city`    
+     - 
+   * - Mailing address state
+     - :code:`mailing_address_state`  
+     - 
+   * - Mailing address ZIP code
+     - :code:`mailing_address_zipcode`
+     - 
+   * - Social security number 
+     - :code:`ssn`
+     - Individual Taxpayer Identification Number (ITIN) if no SSN
+   * - Income 
+     - :code:`income`
+     - 
+   * - Employer ID 
+     - :code:`employer_id`
+     -  
+   * - Employer Name 
+     - :code:`employer_name`
+     - 
+   * - Employer street number
+     - :code:`employer_street_number`
+     - 
+   * - Employer street name
+     - :code:`employer_street_name`
+     - 
+   * - Employer unit number
+     - :code:`employer_unit_number`
+     - 
+   * - Employer city
+     - :code:`employer_city`    
+     - 
+   * - Employer state
+     - :code:`employer_state`  
+     - 
+   * - Employer ZIP code
+     - :code:`employer_zipcode`
+     - 
+   * - Joint filer first name
```

Comment:
Is there a reason why we say "joint filer" in this table but "spouse" in the column name itself? I guess "spouse" is a little more confusing (you could have a spouse that doesn't appear in your 1040, if you file separately) so we opted for a more precise term but didn't want to bug the engineers?

Reply:
I guess so - I think that's what was decided in this [Slack thread ](https://ihme.slack.com/archives/C02KUQ9LX32/p1688405675889299)